id-library-showcase/src/main.rs

```diff
@@ -155,13 +220,22 @@ struct ProveAttributeInRange {
     lower: ExampleAttribute,
     #[structopt(
         long = "randomness",
-        help = "Path to file containing randomness used to produce to commitment."
+        help = "Path to file containing randomness used to produce to commitment.",
+        required_unless = "wallet",
+        conflicts_with = "wallet"
     )]
-    randomness: PathBuf,
-    #[structopt(long = "global", help = "Path to file with global context.")]
-    global: PathBuf,
+    randomness: Option<PathBuf>,
+    #[structopt(
+        long = "wallet",
+        help = "Path to file mobile wallet export.",
+        required_unless = "randomness",
+        conflicts_with = "randomness"
+    )]
+    wallet: Option<PathBuf>,
     #[structopt(long = "proof-out", help = "Path to output proof to.")]
     out: PathBuf,
+    #[structopt(long = "grpc")]
```

Comment:
We call this `--node` in some of the rust-sdk examples, so I suggest you rename the argument to that.